ambry-admin/src/main/java/com.github.ambry.admin/AdminSecurityService.java

```diff
@@ -187,7 +187,17 @@ private void setGetBlobResponseHeaders(RestResponseChannel restResponseChannel,
         restResponseChannel.setHeader("Content-Disposition", "attachment");
       }
     }
-    if (blobProperties.isPrivate()) {
+    setCacheHeaders(blobProperties.isPrivate(), restResponseChannel);
+  }
+
+  /**
+   * Sets headers that provide directions to proxies and caches.
+   * @param isPrivate whether the blob is private.
+   * @param restResponseChannel the channel that the response will be sent over
+   * @throws RestServiceException if there is any problem setting the headers
+   */
+  private void setCacheHeaders(boolean isPrivate, RestResponseChannel restResponseChannel) throws RestServiceException {
+    if (isPrivate) {
       restResponseChannel.setHeader(RestUtils.Headers.EXPIRES, new Date(0));
```

Comment:
I understand this is old code, but why are private blobs handled differently as far as EXPIRES is concerned, and what is the implication of setting EXPIRES to a time in the past?